_pages/file-a-complaint.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: File a Complaint
-description: "The Americans with Disabilities Act provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency.  This page outlines the steps to get you started."
+description: "The ADA provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency. This page outlines the steps to do so."
```

Comment:
supernit: me being [foolishly consistent](https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)

```suggestion
description: The ADA provides an important tool to fight discrimination: filing a complaint with an appropriate federal agency. This page outlines the steps to do so.
```

Reply:
I added the quotes because otherwise the ':' breaks the field

Reply:
ah, nice, makes sense